python/destinations/S3/README.md

```diff
@@ -1,21 +1,26 @@
-# MQTT
+# S3
```

Comment:
Needs to be more descriptive. S3 Sink? Destination? etc?

Reply:
72bd817d52a92411ba5d7db58b707c35e779aa23